sdk/core/azure-core-serializer-json-jackson/src/test/java/com/azure/core/serializer/json/jackson/JacksonPropertyNameTests.java

```diff
@@ -98,10 +97,35 @@ public void testPropertyNameOnMethodName() throws NoSuchMethodException {
             public String getHotelName() {
                 return hotelName;
             }
+            public void setHotelName(String hotelName) {
+                this.hotelName = hotelName;
+            }
         }
 
-        Method m = LocalHotel.class.getDeclaredMethod("getHotelName");
-        assertEquals("getHotelName", serializer.convertMemberName(m));
+        Method getterM = LocalHotel.class.getDeclaredMethod("getHotelName");
+        assertEquals("hotelName", serializer.convertMemberName(getterM));
+        Method setterM = LocalHotel.class.getDeclaredMethod("setHotelName", String.class);
+        assertEquals("hotelName", serializer.convertMemberName(setterM));
+    }
+
+    @Test
+    public void testPropertyNameOnMethodNameWithoutGetSet() throws NoSuchMethodException {
+        class LocalHotel {
+            String hotelName;
+
+            public String hotelName1() {
+                return hotelName;
+            }
+
+            public void hotelName2(String hotelName) {
+                this.hotelName = hotelName;
+            }
+        }
+
+        Method getterM = LocalHotel.class.getDeclaredMethod("hotelName1");
```

Comment:
Add some tests where the property name also contains `get` and `set`.

Reply:
More test coverage now